packages/x-date-pickers-pro/src/MultiInputDateTimeRangeField/MultiInputDateTimeRangeField.tsx

```diff
@@ -0,0 +1,147 @@
+import * as React from 'react';
+import Stack, { StackProps } from '@mui/material/Stack';
+import TextField, { TextFieldProps } from '@mui/material/TextField';
+import Typography, { TypographyProps } from '@mui/material/Typography';
+import { styled, useThemeProps } from '@mui/material/styles';
+import { useSlotProps } from '@mui/base/utils';
+import { MultiInputDateTimeRangeFieldProps } from './MultiInputDateTimeRangeField.types';
+import { useMultiInputDateTimeRangeField } from '../internal/hooks/useMultiInputRangeField/useMultiInputDateTimeRangeField';
+
+const MultiInputDateTimeRangeFieldRoot = styled(
```

Comment:
By the way, I find this part very ugly (including on the existing component)

If you have a better idea to create a slot with props passed only to the default slot.